src/autora/doc/pipelines/main.py

```diff
@@ -15,6 +18,27 @@
     format="%(asctime)s %(levelname)s %(module)s.%(funcName)s(): %(message)s",
 )
 logger = logging.getLogger(__name__)
+nltk.download("wordnet")
+
+
+def evaluate_documentation(predictions: List[List[str]], references: List[str]) -> Tuple[float, float]:
+    # Tokenize predictions and references
+    tokenized_predictions = [pred[0].split() if pred else [] for pred in predictions]
```

Comment:
Can you add a TODO comment and log an issue to generalize this to multiple predictions? I.e. average over `pred` vs just `pred[0]`

Reply:
Done